src/server/scripts/Outland/zone_blades_edge_mountains.cpp

```diff
@@ -1013,6 +1014,53 @@ class spell_oscillating_field : public SpellScriptLoader
         }
 };
 
+enum Marmot
+{
+    SPELL_COAX_MARMOT = 38544
+};
+
+struct npc_marmot : public PossessedAI
+{
+    npc_marmot(Creature* creature) : PossessedAI(creature) { }
+
+    void OnCharmed(bool apply) override
+    {
+        if (apply)
+            me->SetLevel(me->GetCharmerOrOwner()->GetLevel());
```

Comment:
This depends on the absence of a SummonProperties flag (it should be done automatically (NYI))